templates/job-filters.php

```diff
@@ -36,6 +36,13 @@
 			<input type="text" name="search_location" id="search_location" placeholder="<?php esc_attr_e( 'Location', 'wp-job-manager' ); ?>" value="<?php echo esc_attr( $location ); ?>" />
 		</div>
 
+		<?php if( apply_filters( 'job_manager_job_filters_show_remote_position', get_option('job_manager_enable_remote_position', true ), $atts ) ) : ?>
```

Comment:
Shouldn't the default value for `get_option('job_manager_enable_remote_position', true )` be `false` here?

Reply:
So the reason I left it as true is because the original release with the `remote_position` field added had no way to disable it, so I was matching the actual setting default value (which is remote position enabled by default).

Reason being, say someone has installed the original version that added the `remote_position` field but never went into settings and saved them at all.  When they update to this release, the field will not show on the form even though they "assume" it's enabled based on the default value.

Figured users would want this field to show on the filters by default, if they want to disable it can be done via the settings or filter if you specifically don't want to show on the filters page (but default it shows)

Reply:
Good call. I agree.